src-gui/src/main/java/org/openkilda/integration/service/FlowsIntegrationService.java

```diff
@@ -200,6 +201,33 @@ public FlowPath rerouteFlow(String flowId) {
         }
     }
 
+    /**
+     * Reroute Y-flow.
+     *
+     * @param yFlowId y-flow id
+     * @return YFlowRerouteResult
+     */
+    public YFlowRerouteResult rerouteYFlow(String yFlowId) {
```

Comment:
do think require `yFlowId` to be non-null will be useful here?

Reply:
fixed